tools/rapids-prompt-local-repo-config

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# A utility script that prompts user to setup repository information in
+# local environments
+
+if [ "${CI:-false}" = "false" ]; then
+  rapids-echo-stderr "Local run detected."
+  rapids-echo-stderr "NVIDIA VPN connectivity is required to download workflow artifacts."
+
+  if [ -z "${RAPIDS_BUILD_TYPE:-}" ]; then
```

Comment:
Is there any point in the `:-` if there's no default afterwards?

Reply:
Same question applies to the other vars here as well.

Reply:
I suppose it initializes the variable to an empty string? Not sure if it's necessary

Reply:
This is necessary because of the `set -u` option that we use at the top of each script.

`set -u` treats unset variables as errors ([src](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html)).

If we didn't include the `:-` syntax in `"${RAPIDS_BUILD_TYPE:-}"`, the script would error that `RAPIDS_BUILD_TYPE` is not set.

Using `:-` to initialize it as an empty string if it's not already set allows us to make the conditional statement work with the `set -u` option.

Reply:
Ah OK got it.